CHANGELOG.md

```diff
@@ -31,6 +31,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   ([#376](https://github.com/ipetkov/crane/pull/376))
 * Replaced various internal usages of `runCommandLocal` with `runCommand` for
   more optimal behavior when downloading cached artifacts
+* Fixed a bug when a git dependency crate relies on reading non-TOML metadata
+  from Cargo.toml at build time.
+  ([#407](https://github.com/ipetkov/crane/pull/407))
```

Comment:
This should go in a fresh `### Changed` section after `## Unreleased` (since the 0.14.0 tag has already been published) :smile:

Reply:
Ah, I see what you mean!  Updated the changelog to fix.